scout/utils/acmg.py

```diff
@@ -1,4 +1,9 @@
 # coding=UTF-8
+from flask import flash
```

Comment:
```suggestion
```

Reply:
Ahem, right - decided on the js path instead. 👍